.travis.yml

```diff
@@ -101,7 +101,8 @@ script:
       --browser-arg-int build  "$TRAVIS_BUILD_NUMBER" \
       --browser-arg-int idleTimeout 300 \
       --browser-arg name "$TRAVIS_REPO_SLUG $TRAVIS_BRANCH $TRAVIS_COMMIT" \
-      --browser-arg browser "$BROWSER"
+      --browser-arg browser "$BROWSER" \
+      --browser-arg platform "Linux"
```

Comment:
revert this change and rebase since this is in snovault now.

Reply:
I removed the two commit related to this and then rebased on master.